core/src/main/scala/Monitoring.scala

```diff
@@ -60,10 +60,10 @@ trait Monitoring {
       buf: TBuffer[Option[I],O], costive: Boolean = false
     ): (Key[O], Task[I => Task[Unit]]) = {
       val k = keyMod(Key[O](name, units, description))
-      (k, topic(k, costive)(buf))
+      (k, topicWithKey(k, costive)(buf))
     }
 
-  def topic[I,O](
+  def topicWithKey[I,O](
```

Comment:
So, in 2.11 you apparently can't have overloaded methods with default arguments, so I just renamed one of the overloads. You might prefer a different approach.